crates/net/network/src/network.rs

```diff
@@ -431,7 +440,7 @@ pub(crate) enum NetworkHandleMessage {
     /// Disconnects a connection to a peer if it exists, optionally providing a disconnect reason.
     DisconnectPeer(PeerId, Option<DisconnectReason>),
     /// Adds a new listener for `NetworkEvent`.
-    EventListener(UnboundedSender<NetworkEvent>),
+    EventListener(EventListeners<NetworkEvent>),
```

Comment:
these docs are no longer correct.

this is a bit of a footgun because this allows obtaining a sender.

do we even still need this? because this can be configured without going through the channel:

https://github.com/paradigmxyz/reth/blob/c40088741ab2d80887437d59c0ad52a21b6fe57b/crates/net/network/src/manager.rs#L249-L268

so I think this is obsolete now?

Reply:
great catch! absolutely, already removed :+1: